build/automation/var/profile/dev.mk

```diff
@@ -8,12 +8,12 @@ ENV := dev
 SERVICE_PREFIX := $(PROJECT_GROUP_SHORT)-$(PROJECT_NAME_SHORT)-$(ENVIRONMENT)
 DEPLOYMENT_SECRETS := $(PROJECT_ID)/deployment
 
-K8S_TTL :=
+K8S_TTL_LENGTH := 2 days
```

Comment:
For consistency it is better, as well as for practicality it is safer to use the `recursively expanded variables` by using the `=` assignment operator. Do we think we can change it?

See for more details https://www.gnu.org/software/make/manual/html_node/Flavors.html#Flavors

Reply:
Will do